src/Components/GrowthChart/GrowthChartBuilder/GrowthChartBuilder.tsx

```diff
@@ -51,7 +51,7 @@ export const GrowthChartBuilder = ({
                 ticks: { padding: 18 },
             },
         },
-        animation: { onProgress: (chartAnimation: any) => annotateLineEnd(chartAnimation) },
+        animation: { onComplete: (chartAnimation: any) => annotateLineEnd(chartAnimation) },
     };
```

Comment:
You can add the onProgress as well to make the annotation appear on progress as well.
```suggestion
        animation: {
            onComplete: (chartAnimation: any) => annotateLineEnd(chartAnimation),
            onProgress: (chartAnimation: any) => annotateLineEnd(chartAnimation),
        },
```